shared/feature/feedconfirmation/src/main/java/com/epmedu/animeal/feedconfirmation/domain/ValidateFeedingPointAvailableUseCase.kt

```diff
@@ -0,0 +1,14 @@
+package com.epmedu.animeal.feedconfirmation.domain
+
+import com.epmedu.animeal.feeding.domain.model.enum.AnimalState
+import com.epmedu.animeal.feeding.domain.repository.FeedingPointRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import javax.inject.Inject
+
+class ValidateFeedingPointAvailableUseCase @Inject constructor(private val feedingPointRepository: FeedingPointRepository) {
```

Comment:
This usecase won't help if two users book the same feeding point at the same time. 
Let's just handle the exception we get from backend on trying to start feeding.